### PR TITLE
Jared glanceupdate

### PR DESCRIPTION
**Pull request by jmimico**
Hey all, please review my blog post

### CODE REVIEW

File: _posts/jared_baker/2016-11-18-Glance-Images.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+* Log script output to a file that is handled by logstash
+* By default run as a check & notifier via cron but be able to run interactively to update with added syntax (--update)
+
+## The Bash script
+[Available here](https://github.com/CancerCollaboratory/infrastructure/blob/master/utils/image_refresh.sh)
+
+#### Disclaimer
+I come from a IP Networking & technical support background. I don't claim to be of a developers mindset and clearly you will see that in my rudimentary implementation of automating this task. I encourage you to take this script and make it your own.
+
+#### Prerequisites
```
Comment (Fgerthoffert) [inline _posts/jared_baker/2016-11-18-Glance-Images.md]: I would move pre-requisites just after what does it do

File: _posts/jared_baker/2016-11-18-Glance-Images.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+* Alert Cloud admins via email when a new image is available
+* Perform checksums along the way to ensure valid images are being downloaded
+* Maintain a friendly and consistent image name in OpenStack (ie "CentOS 7 - latest") so users of the cloud can use predictable names in their automated workflows.
+* Log script output to a file that is handled by logstash
+* By default run as a check & notifier via cron but be able to run interactively to update with added syntax (--update)
+
+## The Bash script
+[Available here](https://github.com/CancerCollaboratory/infrastructure/blob/master/utils/image_refresh.sh)
+
+#### Disclaimer
```
Comment (Fgerthoffert) [inline _posts/jared_baker/2016-11-18-Glance-Images.md]: I would move disclamer just before the link to the script

File: _posts/jared_baker/2016-11-18-Glance-Images.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+I come from a IP Networking & technical support background. I don't claim to be of a developers mindset and clearly you will see that in my rudimentary implementation of automating this task. I encourage you to take this script and make it your own.
+
+#### Prerequisites
+* [openstack client](https://github.com/openstack/python-openstackclient)
+* [gmail_alert.sh](https://gist.github.com/superdaigo/3754055/)
+
+## Wish List
+* Make the script easier to maintain by using for loops
```
Comment (Fgerthoffert) [inline _posts/jared_baker/2016-11-18-Glance-Images.md:75]: Could you add a small introductory phrase before the wish list (i.e. why wasn't the wish list part of the implementation)

**Comment by Fgerthoffert** [comment]
👍

**Comment by gmihaiescu** [comment]
I would add a few words about how the script is used (cron job running daily), and the reason why we had to create the script in the first place, instead of using an existing solution.

**Pull request merged**